.github/workflows/release.yml

```diff
@@ -58,13 +62,14 @@ jobs:
         with:
           gradle-version: wrapper
           arguments: |
+            shadowExecutableJar
             publishAllPublicationsToGitHubRepository
       - name: 'GitHub Release'
         id: create_release
         uses: actions/create-release@v1
         with:
           tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
+          release_name: Release ${{ env.RELEASE_VERSION }}
```

Comment:
not `GITHUB_ENV.RELEASE_VERSION`?

Reply:
@kgevorkyan No, see https://docs.github.com/en/actions/learn-github-actions/variables

`${{ env.RELEASE_VERSION }}` means "expand the string with the value of `RELEASE_VERSION` environment variable".